Source/Editor/DualityEditor/Forms/NewProjectDialog.Designer.cs

```diff
@@ -63,6 +63,7 @@ private void InitializeComponent()
 			// 
 			this.folderView.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(196)))), ((int)(((byte)(196)))), ((int)(((byte)(196)))));
 			this.folderView.BorderStyle = System.Windows.Forms.BorderStyle.FixedSingle;
+			this.folderView.ColumnHeaderHeight = 0;
```

Comment:
This line wouldn't have been necessary for this commit - it's not crucial, but always best to keep out unintended changes :)